extensions/logging-sentry/deployment/src/test/java/io/quarkus/logging/sentry/SentryLoggerCustomTest.java

```diff
@@ -21,12 +24,13 @@
     @Test
     public void sentryLoggerCustomTest() {
         final Handler sentryHandler = getSentryHandler();
+        final SentryOptions options = HubAdapter.getInstance().getOptions();
```

Comment:
Ah cool, we can access the `SentryOptions`!

Reply:
Yes! We have relaxed a bit the API in 4.x